Add gargammel slim

genomewalker opened the pull request:
Renamed branch to gargammel-slim. Coming from here: https://github.com/bioconda/bioconda-recipes/pull/24243

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

genomewalker commented:
Ping @bgruening

genomewalker commented:
@BiocondaBot please add label

bwlang commented:
@BiocondaBot please fetch artifacts

bwlang commented on recipes/gargammel-slim/meta.yaml:
```diff
@@ -0,0 +1,51 @@
+
+
+source:
+  - url: https://github.com/grenaud/libgab/archive/d2bbab7d88fbfdaa0ce57a26dc908d2fc66bf656.tar.gz
```
seems a bit strange that this is not released... but i guess there's no harm to reproducibility with this snapshot

bwlang reviewed (approved):
some of these patches looks like they would be useful upstream, but I don't see a problem with them and the build works for me.

genomewalker commented:
> some of these patches look like they would be useful upstream, but I don't see a problem with them and the build works for me.

I am adding some of the changes in this recipe, to the original gargammel recipe. I will try to include some of the patches upstream, and ask for a libgab release to the authors.

Thanks

genomewalker commented:
Hi @bwlang 
is anything else needed for merging this branch?

Thanks
Antonio

bwlang commented:
sorry -  i thought you were planning the changes you mentioned for this PR.
@BiocondaBot  please merge

genomewalker commented:
> sorry - i thought you were planning the changes you mentioned for this PR.

No problem :-) 

I added some of the changes I implemented in the slim recipe to the complete gargammel recipe https://github.com/bioconda/bioconda-recipes/pull/24260

When I have some time I will try to add the patches upstream

Many thanks!
Antonio

dpryan79 commented:
@bioconda-bot please merge

dpryan79 commented:
@bioconda-bot please merge
